vlib/v/help/build/build-c.txt

```diff
@@ -180,36 +180,29 @@ see also `v help build`.
       more performant without autofree.
 
    -gc <mode>
-      Use and link an optional garbage collector. Only the Boehm–Demers–Weiser
-      garbage collector is supported currently with the following sub-options:
+      V uses a garbage collector by default. Only the Boehm–Demers–Weiser garbage collector
+      is supported currently with the following sub-options:
 
-      `-gc boehm` ........... selects the default mode for the architecture
+      `-gc boehm` ........... use default garbage collector (same as not specifying option)
       `-gc boehm_full` ...... full garbage collection mode
       `-gc boehm_incr` ...... incremental/generational garbage collection mode
       `-gc boehm_full_opt` .. optimized full garbage collection mode
       `-gc boehm_incr_opt` .. optimized incremental/generational garbage collection mode
       `-gc boehm_leak` ...... leak detection mode
+      `-gc none` ............ no garbage collector
 
-      You need to install a `libgc-dev` package first, or install it manually from:
-
-          https://github.com/ivmai/bdwgc
```

Comment:
I think it is nice to have it mentioned here that we use <https://github.com/ivmai/bdwgc> .

Reply:
Mentioned, yes, but it shouldn't be necessary to install it.

Reply:
... and it's mentioned in the new paragraph at the bottom (which I'm about to commit.